etc/makefiles/sketch.mk

```diff
@@ -167,9 +167,20 @@ else
 local_cflags_property =
 endif
 
-compile:
+# If you set KALEIDOSCOPE_LOCAL_LIB_DIR to the name of a directory, 
+# all of the Arduino libraries inside that directory should be used 
+# in preference to any library with the same name further dow the search path
+
+ifdef ($(KALEIDOSCOPE_LOCAL_LIB_DIR))
```

Comment:
Two minor errors here. It should be:
```
ifneq ($(KALEIDOSCOPE_LOCAL_LIB_DIR),)
_arduino_local_libraries_prop =  --libraries "${KALEIDOSCOPE_LOCAL_LIB_DIR}"
```
(`ifneq`, not `ifdef`, and `--libraries`, not `-libraries`)

Reply:
Wit those minor adjustments, it all works well for me.  Thanks!